packages/@react-stately/color/src/Color.ts

```diff
@@ -10,13 +10,13 @@
  * governing permissions and limitations under the License.
  */
 
-/** A list of supported color formats. */
-type ColorFormat = 'hex' | 'hexa' | 'rgb' | 'rgba' | 'hsl' | 'hsla' | 'hsb' | 'hsba';
+import {
+  ColorFormat,
+  ColorChannel,
+  Color as ColorType,
```

Comment:
I wonder if it's okay to leave a `Color` interface in `@react-types/color` and `Color` class in `@react-stately/color`

Reply:
Yep, that's correct!